wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,589 @@
+---
+tags:
+- UK Community Cup 7
+- UKCC 7
+- UKCC 7
+---
+
+# UK Community Cup 7
+
+![UKCC 7 Logo](img/banner.png)
+
+The 7th edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.
+
+# Contents
```

Comment:
Don't think this whole contents will be very useful, since the site itself has a built-in table of contents in the sidebar.

Reply:
Good point - i'll take it out